src/service_args.py

```diff
@@ -90,6 +91,8 @@ def __add_secondchain_args(self, args: list):
 
     def __customize_service_args(self):
         self.__add_firstchain_args(['--node-key-file', utils.NODE_KEY_PATH])
+        if self._relay_rpc_urls:
+            self.__add_firstchain_args(['--relay-chain-rpc-urls'] + list(self._relay_rpc_urls.values()))
```

Comment:
> Worth knowing is that the URLs from the interface is added to the beginning of the service args so they will be used before any manually added URLs are used.

So, if they're supposed to always be first I think we should make it clear in the docs somewhere that that's the case, to avoid potential confusion. This because setting things manually often means that they're an override and not a fallback.

Perhaps it could a mention in the readme as well as a mention for the `service-args` charm config. What do you say?

Reply:
I'll add something :+1:

Reply:
I added a section about it in the README.